### ISSUE TITLE
questions

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/tensor-something#1]
Hello Arthur,

Its the week-end and I have 3 free days for projects. :)
Do you have some new information about what you said before ? 

On my side, I've been studying Tensorflow and thinking about how we would implement this in real for production context. It's quiet complicated for the production, I think it's too early for me to think about putting it in production.

What I'am sure about is that I'am going to use Tensorflow to build a neural network to process food / dish pictures . Just to give a brief description of what it is : 

From wikipedia : 
"TensorFlow is an open-source software library for dataflow programming across a range of tasks. It is a symbolic math library, and also used for machine learning applications such as neural networks.[3] It is used for both research and production at Google,‍[3]:min 0:15/2:17 [4]:p.2 [3]:0:26/2:17 often replacing its closed-source predecessor, DistBelief. "

Without being ironic, it's possible to implement neural networks with a few lines of python code. That's really cool, no kidding but it requires a good design, so a lot of knowledge about the picture we want to recognize would help ( it will consequently tell us on which data I will train my model and this is REALLY important to know ) 

But to make sure that it is a powerful model (a model that doesn't make mistake in recognition), I need answers to the following questions =D :  

- Dimensions of the picture (let's say a mean dimension, but technically, there's a way to deal with this problem, but if you give me a precise dimension, thats the best)
- What will be on the user's pictures ? Will there be simple ingredients like a picture of some carrots and onions ? Will there be pictures of dishes ? (in that case, its a harder challenge but still possible to do) 

Important : I really need those answers to go forward in this task ! And you'll be surprised that it won't require months to implement it, we just need to know exactly what we want to analyze, what are the results we are aiming (The ultimate list of Groceristar =)  ). I will take care to implement it in a good way, to train the model and to provide the solution, then we will think about put it in production.

If you have any questions, don't hesitate to ask.

Kind regards,
Philippe.

**Comment by atherdon** [comment]
i agree to use TensorFlow. i know that it have a great potential, but i'm not a pro at this thing. 
So i think i need to agree with you and give you all freedom.

I'll reply here in short, and go and find all important information to you and move it into this repository.

* **Dimensions can be different**. But for a first time, i think we can set some sort of a rule, where we work with only with AxB dimensions only. Just tell me what will be ok, or let's pick that size together.

* content types. it also can be different things inside. But, let's make it more simple. You mentioned Ultimate GL list. let's assume that this type will be our basic thing. I'll attach a pdf file() that i use to grab data by my hands :) 
I also will share with you some images, that i grab before. they can give you more information about what content can be presented. I picked different images from pinterest, so i think it can be a good test cases.
So for not - **let's assume that we have text only on our images.**

**Comment by atherdon** [comment]
Tell me if i didn't clarify something

**Comment by philippeTUESIGELEC** [comment]
Hi Arthur ! 

Thanks for the answer.
I'll start to work on pictures of different dimensions on tensorflow. I don't think text on pictures is a great idea. It would be better directly working in a text format using natural language processing (better results).

**Comment by philippeTUESIGELEC** [comment]
Also, I'am working into building a GL List using open data, i'll keep you advised about this soon.

**Comment by atherdon** [comment]
please click on this link in order to add you as collaborator:
https://github.com/GroceriStar/tensor-something/invitations
